testsuite/features/support/client_stack.rb

```diff
@@ -67,17 +67,21 @@ def check_restart(host, node, time_out)
   end
 end
 
-# Extract the OS version by decoding the value in '/etc/os-release'
-# Use VERSION id = false, VERSION_ID otherwise
-def get_os_version(node, id = true)
-  field = id ? 'VERSION_ID' : 'VERSION'
-  os_version_raw, _code = node.run('grep "' + field + '=" /etc/os-release')
+# Extract the OS version and OS family
+# We get these data decoding the values in '/etc/os-release'
+def get_os_version(node)
+  os_family_raw, _code = $proxy.run('grep "^ID=" /etc/os-release')
+  os_family = os_family_raw.strip.split('=')[1]
+  return nil, nil if os_family.nil?
+  os_family.delete! '"'
+
+  os_version_raw, _code = node.run('grep "^VERSION_ID=' + field + '=" /etc/os-release')
```

Comment:
I don't understand, why do we have `field` here?

Reply:
gasp